.babelrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why is the indentation here funky? All indentations should be uniform across all files